src/io/stapled.cr

```diff
@@ -73,7 +73,7 @@ class IO::Stapled < IO
     @writer.write(slice)
   end
 
-  # `Flushes `writer`.
+  # Flushes `writer`.
```

Comment:
`*writer*`

Reply:
That's incorrect. * are used to denote arguments, not properties.

Reply:
But ` doesn't seem correct to me either. * seems more correct to me in this case because it's more related to a variable/parameter.

And currently here: [`stapled.cr:21`](https://github.com/crystal-lang/crystal/blob/master/src/io/stapled.cr#L21)
```cr
# Creates a new `IO::Stapled` which reads from *reader*  and writes to *writer*-
def initialize(@reader : IO, @writer : IO, @sync_close : Bool = false)
```
there's also * used to italicize the parameter. And now here using ` would be a bit incosistent I think.

Reply:
Yes, because here they're arguments, not properties, that's the whole point.

Reply:
My bad, this is correct as-is.